frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/AccountRecoveryResponse.java

```diff
@@ -3,17 +3,5 @@
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
-public class AccountRecoveryResponse {
-
-    @SerializedName("accountRecoveryPermitted")
-    @Expose
-    private boolean accountRecoveryPermitted;
-
-    public AccountRecoveryResponse(boolean accountRecoveryPermitted) {
-        this.accountRecoveryPermitted = accountRecoveryPermitted;
-    }
-
-    public boolean getAccountRecoveryPermitted() {
-        return accountRecoveryPermitted;
-    }
-}
+public record AccountRecoveryResponse(
+        @SerializedName("accountRecoveryPermitted") @Expose boolean accountRecoveryPermitted) {}
```

Comment:
As the field name is the same as the property name, we don't need the `@SerializedName` annotation.

Reply:
We discussed - this actually is currently needed as without further code changes the default serialisation is snake case